mapserver/c2cgeoportal.map.mako

```diff
@@ -76,6 +76,7 @@ MAP
     END
 
     LEGEND
+        KEYSIZE 20 20
```

Comment:
Without:
![gdl_config_default_remote_directory1](https://cloud.githubusercontent.com/assets/660191/15356393/afc64ef2-1cf8-11e6-81db-37f44fae3b58.png)
With :
![gdl_config_default_remote_directory1](https://cloud.githubusercontent.com/assets/660191/15356420/e25e3488-1cf8-11e6-97e4-145824c46bc2.png)

But indeed, keyimage ratio is not correct.